src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2556,93 +2539,183 @@ SpinelNCPInstance::handle_ncp_spinel_callback(unsigned int command, const uint8_
 	process_event(EVENT_NCP(command), cmd_data_ptr[0], cmd_data_ptr, cmd_data_len);
 }
 
-void
-SpinelNCPInstance::address_was_added(const struct in6_addr& addr, int prefix_len)
+bool
+SpinelNCPInstance::should_filter_address(const struct in6_addr &addr, uint8_t prefix_len)
```

Comment:
We can consider it. Currently OT does not filter and pass up all addresses. There may be value for debugging to see all addresses (including RLOC ones) on wpantund.
We were filtering the RLOC based addresses when the `IPV6_ADDRESS_TABLE` was parsed, I added a method to make it more explicit.